src/DB/Query/ShippingRateQuery.php

```diff
@@ -41,6 +41,28 @@ protected function sanitize_value( string $column, $value ) {
 			throw InvalidQuery::cant_set_id( ShippingRateTable::class );
 		}
 
+		if ( 'options' === $column ) {
+			$value = maybe_serialize( $value );
```

Comment:
This isn't fully sanitizing as the name of the function implies. Any data could be inserted once serialized, not just a set of options. Do we need to check here if it's in the format we expect?

Reply:
I don't think we need to check the exact format because that gets a bit outside the responsibility of this class. I originally wanted to define "types" for each of the columns so that we can properly validate and sanitize them but that would make these query/table classes more complicated and push them towards being an ORM.

I added a check to make sure that the provided value is an array and throw an exception otherwise. However, this still sounds like a "validation" and not "sanitization". Even though the line above does a similar check for the `id` column.

Reply:
Yes I agree, none of the classes are really doing sanitization in that function. Renaming it might be clearer but at this state it's probably better to leave it as is.